repo/packages/H/httpbin/0/package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "description": "HTTP Request & Response Service",
```

Comment:
Please add details to the description. You can take it from the example: 
`httpbin is an HTTP Request & Response Service, written in Python with the Flask framework. Basically, it's an HTTP server that responds in predictable ways to special HTTP requests and echoes back a lot of the data given to it, so that a developer can see what HTTP client code is doing`